chunks.go

```diff
@@ -216,6 +220,8 @@ func (w *chunkWriter) WriteChunks(chks ...*ChunkMeta) error {
 		chk.Chunk = nil
 	}
 
+	// XXX: Shouldn't the CRC32 length be included in maxLen?
+	// XXX: Shouldn't w.n be incremented here?
```

Comment:
`w.n` is incremented in every call to `write` as below.
CRC32s are generally never included in the length. The general concept is `<len(data), data, crc(data)>`, which is also found in the index plenty of times.